packages/web-client/src/helpers/space/functions.ts

```diff
@@ -180,6 +180,7 @@ export function buildSpace(
     description: data.description,
     extension: '',
     path: '/',
+    webUrl: data.webUrl,
```

Comment:
can't we name it `privateLink` here as well to avoid the case differentiation in the CopyPrivateLink component? 🤔

Reply:
we could, but thats the raw data from graph, do we really want to coerce it to another name ?

Reply:
oh yeah, quite true we already have private link on that resourceType

Reply:
done :-)